util/package.js

```diff
@@ -1,5 +1,20 @@
 'use strict';
 
+var allStars = require( 'all-stars' );
+var objectAssign = require( 'object-assign' );
+
+/**
+ * Check all-stars db for given person and assign to person if found.
+ *
+ * @param  {Object} person object representing author or maintainer
+ *
+ * @return {Object}        same object given, possibly modified or augmented
+ */
+function getAllStar( person ) {
+  // override properties from all-stars if available
+  return objectAssign( person, allStars( person ) );
+}
```

Comment:
Ok I'm confused.

My first approach explicitly added string properties `npm`, `github`, and `twitter` to the `person` object. But it was using a few `if` statements. You suggested using `extend` or `assign` instead.

My second approach used `object-assign` to implicitly override `name` and `email` and add the following properties to the `person` object:
- `npmUser` string
- `githubUser` string
- `twitter` string
- `id` string
- `emails` array
- `names` array
- `npmUsers` array
- `githubUsers` array
- `twitters` array
- `summary` function
- `toString` function

Which is probably a bit too much.

You mentioned you'd like the `summary` result, which could be represented as one of two strings (instead of a function). I asked which string you'd prefer, and I didn't understand your answer.

How many of the properties listed above would you like? Are you suggesting that the `summary()` function in `all-stars` return an object (instead of a string) with the following properties?
- `name` string
- `email` string
- `npm` string
- `github` string
- `twitter` string

And then merge this with the `person` object via `object-assign`? Something like this?

```js
var allStar = allStars( person );
return allStar ? objectAssign( person, allStar.summary() ) : person;
```